git-gpg-init/README.md

```diff
@@ -9,3 +9,17 @@ description: |
 ---
 
 Alias for https://webinstall.dev/git-config-gpg
+
+### Files
+
+These are the files / directories that are created and/or modified with this
+install:
+
+```text
+~/.config/envman/PATH.env
+~/.local/bin/git-config-gpg
+~/.local/bin/gpg-pubkey-id
+~/.local/bin/gpg-pubkey
+~/.gnupg/pubring.kbx
```

Comment:
Just listing `~/.gnupg/` is fine as that's the config dir and there will be lots of things there over time.

Reply:
Noted and updated.